noetic/distribution.yaml

```diff
@@ -34,11 +34,27 @@ repositories:
       version: master
     status: maintained
   bond_core:
+    doc:
+      type: git
+      url: https://github.com/ros/bond_core.git
+      version: kinetic-devel
```

Comment:
Looks like `bond_core` has a `noetic-devel` branch. Did you mean to use that instead?

Reply:
That is now vestigial, a separate branch will not be required to support both noetic and melodic.